### DISCUSSION TITLE
How to prevent Buttons from getting focus

### DISCUSSION THREAD

**Discussion by luxi78** [quasarframework/quasar#12110] [General - Components / Directives / etc] [answered]
On mobile devices with virtual keyboard, the focus needs to be on the textarea at all times for a better user experience. 
But currently, when one of the buttons on the footer is taped, the button gets the focus and the virtual keyboard disappears, how to stop this from happening?
![ezgif](https://user-images.githubusercontent.com/964179/150197161-49ac68ce-e079-4fdb-bf02-0c67dbb06256.gif)

The code for footer looks like this:
```html
<q-footer v-model="showFooter" bordered class="bg-white text-primary">
    <q-toolbar class="bg-black text-white">
        <q-btn  flat   label="ESC" />
        <q-btn flat   icon="mdi-keyboard-tab" />
        <q-btn flat   label="CTRL" />
        <q-btn flat   label="ALT" />
        <q-btn flat   icon="mdi-chevron-down" />
        <q-btn flat   icon="mdi-chevron-up" />
        <q-btn flat   icon="mdi-keyboard-backspace" />
    </q-toolbar>
</q-footer>
```

Thanks for any help!

**Comment by hawkeye64** [comment]
If you don't want them to be focusable via keyboard navigation, then set the `tabindex` property to -1 (`:tabindex="-1"`)

**Reply by luxi78** [reply]
I just want the virtual keyword doesn't disappear when I tap them.

**Comment by rstoenescu** [comment, marked as answer]
Put a ref on the textarea then call focus() on it on the click event for each button

**Reply by luxi78** [reply]
It works, thank you
